src/traces/parcoords/parcoords.js

```diff
@@ -121,8 +121,14 @@ function model(layout, d, i) {
         dimensions = trace.dimensions,
         width = layout.width;
 
+    var cs = !!trace.line.colorscale && Lib.isArray(trace.line.color);
+    var lineColor = cs ?
+        trace.line.color :
+        Array.apply(0, Array(trace.dimensions.reduce(function(p, n) {return Math.max(p, n.values.length);}, 0))).map(function() {return 0.5;});
+    var lineColorScale = cs ? trace.line.colorscale : [[0, trace.line.color], [1, trace.line.color]];
+
```

Comment:
Hmm. I would prefer not. Per-datum non-pixel related computations should be in the `calc` step.